Sidney-/slugtrade
# Edit item

New functionality:
- New page for editing closet items
- Closet items on the profile are clickable

How to test:
- Login and go to the profile page, then click on one of the closet items. Make sure that you get redirected to /edit_closet_item/
- Make sure that the page looks good
- Try to edit all of the item fields (play around with the images a bit. They have a lot of interconnected logic on both the front and backend so if you find a bug, I bet it will be there)
- Go to someone else's profile through /users and make sure that when you click on a closet item, you don't get redirected
- Logout and try to access the edit_closet_item page and make sure that you get redirected to the error page

Notes:
- The category bug is not fixed yet, so don't try to select any of the broken categories
